net/ser2net/files/etc/init.d/ser2net

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2017 OpenWrt.org
+
+START=75
+STOP=10
+CONFIGFILE="/tmp/ser2net.conf"
+
+USE_PROCD=1
+PROG=/usr/sbin/ser2net
+
+start_service() {
+	config_load 'ser2net'
+
+	local enabled
+	config_get_bool enabled config 'enabled' '0'
+	[ "$enabled" -gt 0 ] || return 1
```

Comment:
You want the enable to be on the use of the UCI conversion.  There's still a gain to be had from an functional init script  with the existing /etc/ser2net.conf file.  particularly when you'll never be able to match the depth of availlable config in the UCI model. Like https://github.com/openwrt/packages/blob/master/net/mosquitto/files/etc/init.d/mosquitto#L186-L191

Reply:
So what you meant is:
if enabled: the /etc/config/ser2net config file gets parsed to /tmp/ser2net.conf.
if disabled: the /etc/ser2net.conf file is used.

Is my assumption correct?

Reply:
yes, that's what I was talking about at least.

Reply:
Seems feasible to me. Do I have to provide a default /etc/ser2net.conf file or can I just check for its existence and fail if not found?

Reply:
well, the package is already installing one, so I wouldn't bother doing any further checking.

Reply:
Ok, agreed.

Reply:
Unless the user explicitly removes it...

Reply:
@pprindeville What if they remove the package itself?! where do you stop?

Reply:
Let's not go over the top.

For someone concerned about security and not having a "fail open" policy, removing defaults to stop a service from running is good security practice.